moveos/moveos-types/src/h256.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+pub use ethereum_types::H256;
+use tiny_keccak::{Hasher, Sha3};
+
+pub const LENGTH: usize = 32;
```

Comment:
The difference of this method is hash[.32], the old method is hash[..], is it for compatibility with the Ethereum chain?

Reply:
Old HashValue also hash[.32].